CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# 0.2.1
+
+Fixed:
+
+* Fixed an issue where Node drains would hang indefinitely on StatefulSet Pods ([#168]), ([#179])
+* Added more restrictive checking of TokenReviewStatus during apiserver auth
+
+Added:
+
+* Added support for IPv6 cluster ([#178])
+
+[#168]: https://github.com/bottlerocket-os/bottlerocket-update-operator/issues/168
+[#179]: https://github.com/bottlerocket-os/bottlerocket-update-operator/issues/179
+[#178]: https://github.com/bottlerocket-os/bottlerocket-update-operator/issues/178
```

Comment:
nit: In the main Bottlerocket changelog, we reference the PRs instead of the issues for each fix/feature. Not sure if we want to keep that consistent here

Reply:
I think we should reference PRs in future.